api/v1/linstorcluster_types.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// LinstorClusterSpec defines the desired state of LinstorCluster
+type LinstorClusterSpec struct {
+	// Repository used to pull workload images.
+	// +kubebuilder:validation:Optional
+	Repository string `json:"repository,omitempty"`
+
+	// NodeSelector selects for which nodes LinstorSatellite resources will be created.
```

Comment:
```suggestion
	// NodeSelector selects which nodes LinstorSatellite resources will be created for.
```

Or:

```suggestion
	// NodeSelector selects the nodes for which LinstorSatellite resources will be created.
```

Or a bit less technical but perhaps more meaningful for the user:

```suggestion
	// NodeSelector selects the nodes on which LINSTOR Satellites will be deployed.
```

Reply:
Thanks, chose the last one.